controllers/apps/operations/upgrade.go

```diff
@@ -181,3 +159,52 @@ func (u upgradeOpsHandler) getClusterComponentVersionMap(ctx context.Context,
 	}
 	return components, nil
 }
+
+func (u upgradeOpsHandler) getCompOpsListByClusterVersion(opsRes *OpsResource) []appsv1alpha1.ComponentOps {
```

Comment:
Why the method name contains "ByClusterVersion" suffix. Doesn't see any ClusterVersion related logics here.